manifests/apache_exporter.pp

```diff
@@ -103,6 +103,7 @@
   Boolean $export_scrape_job              = false,
   Stdlib::Port $scrape_port               = 9117,
   String[1] $scrape_job_name              = 'apache',
+  Optional[Hash] $scrape_job_labels                                  = {},
```

Comment:
any chance to enforce the type in the hash? Also it doesn't need to be optional, if the default is an empty hash.

Reply:
i guess it's something like `label = { foo: bar, ...}`, so how is that expressed in a type?